redis_lru/lru.py

```diff
@@ -24,11 +24,13 @@ def __init__(self,
                  max_size=2 ** 20,
                  default_ttl=15 * 60,
                  key_prefix='RedisLRU',
-                 clear_on_exit=False):
+                 clear_on_exit=False,
+                 exceptions=None):
```

Comment:
the name `exceptions` just like error in Python,  users have probably misunderstood.
please replace it with  `exclude_values`

Reply:
Yes, it's better. I renamed it.